model/config.go

```diff
@@ -489,6 +491,21 @@ func (o *Config) SetDefaults() {
 		*o.SqlSettings.QueryTimeout = 30
 	}
 
+	if o.FileSettings.EnableClientSideEncryption == nil {
+		o.FileSettings.EnableClientSideEncryption = new(bool)
+		*o.FileSettings.EnableClientSideEncryption = false
+	}
+
+	if *o.FileSettings.EnableClientSideEncryption {
+		if o.FileSettings.ClientSideEncryptionKey == nil || *o.FileSettings.ClientSideEncryptionKey == "" {
+			o.FileSettings.ClientSideEncryptionKey = new(string)
+			*o.FileSettings.ClientSideEncryptionKey = NewRandomString(32)
+		}
+	} else {
+		o.FileSettings.ClientSideEncryptionKey = new(string)
```

Comment:
@hmhealey I added that change, but it looks like master is currently broken.  Will this pr automatically re-build when master is fixed, or do I need to somehow kick it back off?